aries-site/src/examples/components/checkbox/CheckBoxValidationExample.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import { Box, CheckBox, Form, FormField } from 'grommet';
+
+export const CheckBoxValidationExample = () => {
+  const defaultErrorMessage = 'Check a checkbox/toggle to resolve this error.';
+  const [message, setMessage] = useState(defaultErrorMessage);
+
+  return (
+    <Box gap="large" direction="row">
+      <Box gap="medium">
+        <Form>
+          <FormField
+            name="required-field"
+            label="Label"
+            htmlFor="required-field"
+            required
+            error={message}
+          >
+            <CheckBox
+              onChange={event => {
+                if (event.target.checked) {
+                  setMessage('');
+                } else {
+                  setMessage(defaultErrorMessage);
+                }
+              }}
+              label="Validation"
+            />
+          </FormField>
+        </Form>
+      </Box>
+      <Box gap="medium">
+        <Form>
+          <FormField
+            name="required-field"
+            label="Label"
+            htmlFor="required-field"
```

Comment:
Need unique id. Cannot have multiple elements with same id
```suggestion
            htmlFor="required-field-2"
```